jobs/cluster/rotate-log-access-key/Jenkinsfile

```diff
@@ -43,7 +43,7 @@ node('openshift-build-1') {
                     ],
                     [
                         $class: 'hudson.model.StringParameterDefinition',
-                        defaultValue: 'pep@redhat.com',
+                        defaultValue: 'jupierce@redhat.com,pep@redhat.com',
```

Comment:
This change is unrelated to the topic of the PR, but thought I'd hijack the PR a bit to include just this minor fix...